src/store-modules/active-media-store.js

```diff
@@ -0,0 +1,39 @@
+import {
+  SET_ACTIVE_MEDIA_ITEM,
+  UNSET_ACTIVE_MEDIA_ITEM,
+} from './mutation-types'
+
+/**
+ * Stores information about the active media item.
+ * @type {import('./types').ActiveMediaState}
+ */
+const state = {
+  activeMediaType: null,
+  activeMediaId: null,
```

Comment:
Small stylistic question, do we need to be this verbose? `activeMediaType` vs. `type` for example. Is the context of the ActiveMediaStore enough? `</bikeshedding>`

Reply:
Since our store modules are not namespaced, I'm concerned that using single-word names like `type` and `id` can be 
- too ambiguous to know what store they represent
- easy to conflict if used by some other store

Reply:
fine with me 😄

Reply:
We might be able to switch to smaller names after #205.